docs/src/extend/plugins.md

```diff
@@ -116,12 +122,13 @@ If the example plugin above were called `eslint-plugin-myPlugin`, the `myConfig`
 
 ```
 
-**Note:** Please note that configuration will not enable any of the plugin's rules by default, and instead should be treated as a standalone config. This means that you must specify your plugin name in the `plugins` array as well as any rules you want to enable that are part of the plugin. Any plugin rules must be prefixed with the short or long plugin name. See [Configure Plugins](../use/configure/plugins#configure-plugins) for more information.
+ESLint configuration files do not enable any of the plugin's rules by default. You must specify your plugin name in the `plugins` array as well as any rules you want to enable that are part of the plugin. Any plugin rules must be prefixed with the short or long plugin name. To learn more, refer to [Configure Plugins](../use/configure/plugins#configure-plugins).
```

Comment:
The first sentence was referring to configs in plugins, not end user config files.

Reply:
i'm not 100% sure if i understood what you meant there, but i think i refactored per your feedback in [bf166a3](https://github.com/eslint/eslint/pull/16862/commits/bf166a3a39148d0d6b3ae579ba9165fdf6e6a1a9). please let me know if you'd like additional changes.